docs/src/theme/VectorStoreTabs.js

```diff
@@ -53,6 +53,13 @@ export default function VectorStoreTabs(props) {
             packageName: "langchain-mongodb",
             default: false,
         },
+        {
+            value: "openGauss",
```

Comment:
In the vectorstoretabs, we're only featuring implementations that are either very well known or else are maintained by companies that we're familiar with